src/include/platform/OTAImageProcessor.h

```diff
@@ -26,9 +26,8 @@
 
 namespace chip {
 
-struct OTAImageProcessorParams
+struct OTAImageProgress
```

Comment:
Curious if this struct is needed at all now?

Reply:
Right, we can probably remove it, too.